### PR TITLE
feat: update hero section images for issue #66

### PR DESCRIPTION
**Pull request by yashcrest**
This pull request updates the images used in the hero section of the home page as per the requirements in issue #66. 

	•	Updated 4 hero images in the /public/images folder.
	•	Modified heroSlider.tsx to use the new images for the homepage slider.

@boltdynamics & @Aaasisss  could you please review and provide feedback?

### CODE REVIEW

File: src/modules/home/heroSlider.tsx [hunk 1]
```diff
@@ -11,10 +11,10 @@ import {
 import Image from "next/image";
 
 const heroImages = [
-  { id: 1, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 1" },
-  { id: 2, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 2" },
-  { id: 3, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 3" },
-  { id: 4, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 4" },
+  { id: 1, imageSrc: "/images/hero-image-1.jpg", imageAlt: "group image 1" },
+  { id: 2, imageSrc: "/images/hero-image-2.jpg", imageAlt: "group image 2" },
+  { id: 3, imageSrc: "/images/hero-image-3.jpg", imageAlt: "group image 3" },
```
Comment (boltdynamics) [inline src/modules/home/heroSlider.tsx:16]: Hey @yashcrest can we please crop the picture frame in the background from hero-image-3? 🙈

Comment (yashcrest) [reply src/modules/home/heroSlider.tsx:16]: lol... sure. 
have cropped out the photo frame from hero-image-3  now 👍 .

Comment (yashcrest) [reply src/modules/home/heroSlider.tsx:16]: I have committed the changes to my branch but I don't see the changes here. Are you able to see it in your end?

Comment (thorn-pod) [inline src/modules/home/heroSlider.tsx:16]: Can we also please reduced the size ? **1.28MB** seems too big.

Comment (yashcrest) [reply src/modules/home/heroSlider.tsx]: @thorn-pod thanks for your input. I have converted the image format to .webp to reduce the file size in the latest commit.

**Review by thorn-pod — APPROVED** [review]

**Review by Aaasisss — APPROVED** [review]
looking good!
let's merged it and think about optimisation later.

**Pull request merged**